keyboards/nixkeyboards/day_off/keymaps/default/keymap.c

```diff
@@ -0,0 +1,60 @@
+      KC_MPLY,    KC_CAPS,   KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_BSLS,  KC_ENT,
+      KC_MNXT,    KC_LSFT,   KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,  KC_BSLS,
+      KC_MPRV,    KC_LCTL,   KC_LALT, KC_LGUI,          KC_SPC,           KC_SPC,  KC_SPC,           KC_RGUI, KC_RALT, KC_APP,  KC_RCTL,  RESET
+    ),
+    
+    [1] = LAYOUT_all(
+                  RESET,     KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,   KC_BSPC,
+                  KC_TAB,    KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC,  KC_BSPC,
+      KC_MPLY,    KC_CAPS,   KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_BSLS,  KC_ENT,
+      KC_MNXT,    KC_LSFT,   KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,  KC_BSLS,
+      KC_MPRV,    KC_LCTL,   KC_LALT, KC_LGUI,          KC_SPC,           KC_SPC,  KC_SPC,           KC_RGUI, KC_RALT, KC_APP,  KC_RCTL,  RESET
+    ),
+    
+    [2] = LAYOUT_all(
+                     KC_GESC,   KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,   KC_BSPC,
+                     KC_TAB,    KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC,  KC_BSPC,
+         KC_MPLY,    KC_CAPS,   KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_BSLS,  KC_ENT,
+         KC_MNXT,    KC_LSFT,   KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,  KC_BSLS,
+         KC_MPRV,    KC_LCTL,   KC_LALT, KC_LGUI,          KC_SPC,           KC_SPC,  KC_SPC,           KC_RGUI, KC_RALT, KC_APP,  KC_RCTL,  RESET
+    ),
+    
+    [3] = LAYOUT_all(
+                     KC_GESC,   KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,   KC_BSPC,
+                     KC_TAB,    KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC,  KC_BSPC,
+         KC_MPLY,    KC_CAPS,   KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_BSLS,  KC_ENT,
+         KC_MNXT,    KC_LSFT,   KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,  KC_BSLS,
+         KC_MPRV,    KC_LCTL,   KC_LALT, KC_LGUI,          KC_SPC,           KC_SPC,  KC_SPC,           KC_RGUI, KC_RALT, KC_APP,  KC_RCTL,  RESET
+    ),
+};
+
+bool encoder_update_user(uint8_t index, bool clockwise) {
+    if (clockwise) {
+        tap_code(KC_VOLU);
+    } 
+    else {
+        tap_code(KC_VOLD);
+    }
+    return false;
+}
```

Comment:
This could use the new [Encoder Map](https://docs.qmk.fm/#/feature_encoders?id=encoder-map) feature.

Reply:
Another time.